# 숙박시설 채널별 평점 변동 정보 DB

<데이터 설명> 숙박시설 채널별 평점 변동 이력 데이터
- 링크: https://www.bigdata-culture.kr/bigdata/user/data_market/detail.do?id=1ea1d390-7ab3-11ed-b3ef-49efc94461a7

- 데이터 소개
    - 전국 숙박시설의 OTA(Online Travel Agency)별 평점 변동 이력 제공
    - 평점이 부여된 OTA, 날짜, 점수 등 기간에 따른 변동 이력 제공

- 데이터 수집기간: 2022-01-01 ~ 2022-09-31 -> 이중 **2024년도 데이터만 진행**
- 본 데이터는 히어로웍스의 숙박시설 수익 관리 시스템 '데이터메니티'를 기반으로 추출된 데이터입니다. 


# 데이터 불러오기

In [53]:
import pandas as pd
df_2024_6 = pd.read_csv('HW_LDGS_OTA_USE_LIST_AVRG_SCORE_VARTION_INFO_202406.csv')
df_2024_12  = pd.read_csv('HW_LDGS_OTA_USE_LIST_AVRG_SCORE_VARTION_INFO_202412.csv')

In [7]:
df_2024_6.head()

LDGS_NM CTPRVN_NM GUGUN_NM OTA_NM  BASE_YEAR  BASE_MT  BASE_DAY  \
0  아르반 호텔     부산광역시     부산진구  AGODA       2023       12         2   
1  아르반 호텔     부산광역시     부산진구  AGODA       2023       12         3   
2  아르반 호텔     부산광역시     부산진구  AGODA       2023       12         4   
3  아르반 호텔     부산광역시     부산진구  AGODA       2023       12         5   
4  아르반 호텔     부산광역시     부산진구  AGODA       2023       12         7   

   AVRG_SCORE_VALUE  REVIEW_REGIST_CO  
0           6.80000               1.0  
1           7.26667               3.0  
2          10.00000               1.0  
3           9.80000               2.0  
4           9.50000               2.0

In [8]:
df_2024_12.head()

LDGS_NM CTPRVN_NM GUGUN_NM OTA_NM  BASE_YEAR  BASE_MT  BASE_DAY  \
0  아르반 호텔     부산광역시     부산진구  AGODA       2024        6         2   
1  아르반 호텔     부산광역시     부산진구  AGODA       2024        6         3   
2  아르반 호텔     부산광역시     부산진구  AGODA       2024        6         4   
3  아르반 호텔     부산광역시     부산진구  AGODA       2024        6         5   
4  아르반 호텔     부산광역시     부산진구  AGODA       2024        6         6   

   AVRG_SCORE_VALUE  REVIEW_REGIST_CO  
0              9.40               2.0  
1              9.50               8.0  
2              8.64               5.0  
3              6.65               4.0  
4              8.70               4.0

In [9]:
df_2024_6[['BASE_YEAR','BASE_MT']].value_counts()

BASE_YEAR  BASE_MT
2024       5          36203
           4          34659
           3          33854
2023       12         32859
2024       1          30950
           2          30037
Name: count, dtype: int64

In [10]:
df_2024_12[['BASE_YEAR','BASE_MT']].value_counts()

BASE_YEAR  BASE_MT
2024       11         71753
           9          68787
           10         68202
           6          34595
           8          33413
           7          26371
Name: count, dtype: int64

- df_2024_6에는 2023년 12월부터 2024년 5월까지 데이터
- df_2024_12에는 2024년 6월부터 11월까지 데이터
- 즉, 두 데이터를 합치면 2023년 12월 ~ 2024년 11월까지 12개월치가 전부 포함된 상태

## 데이터 합치기

In [54]:
import pandas as pd

# CSV 파일 불러오기
df_2024_06 = pd.read_csv('HW_LDGS_OTA_USE_LIST_AVRG_SCORE_VARTION_INFO_202406.csv')
df_2024_12 = pd.read_csv('HW_LDGS_OTA_USE_LIST_AVRG_SCORE_VARTION_INFO_202412.csv')

# 두 데이터프레임 순서대로 합치기
df = pd.concat([df_2024_06, df_2024_12], ignore_index=True)


In [12]:
df.head()

LDGS_NM CTPRVN_NM GUGUN_NM OTA_NM  BASE_YEAR  BASE_MT  BASE_DAY  \
0  아르반 호텔     부산광역시     부산진구  AGODA       2023       12         2   
1  아르반 호텔     부산광역시     부산진구  AGODA       2023       12         3   
2  아르반 호텔     부산광역시     부산진구  AGODA       2023       12         4   
3  아르반 호텔     부산광역시     부산진구  AGODA       2023       12         5   
4  아르반 호텔     부산광역시     부산진구  AGODA       2023       12         7   

   AVRG_SCORE_VALUE  REVIEW_REGIST_CO  
0           6.80000               1.0  
1           7.26667               3.0  
2          10.00000               1.0  
3           9.80000               2.0  
4           9.50000               2.0

# 데이터 정제 (공통)

## 변수명 변경
- 알아보기 쉽게 변수명 변경하기

In [55]:
# 컬럼명 한글로 변경
df.columns = [
    '숙박업명',      # LDGS_NM
    '시도명',        # CTPRVN_NM
    '구군명',        # GUGUN_NM
    '온라인여행사명', # OTA_NM
    '기준년도',      # BASE_YEAR
    '기준월',        # BASE_MT
    '기준일',        # BASE_DAY
    '평점값',        # AVRG_SCORE_VALUE
    '리뷰등록수'     # REVIEW_REGIST_CO
]

In [14]:
df.head()

숙박업명    시도명   구군명 온라인여행사명  기준년도  기준월  기준일       평점값  리뷰등록수
0  아르반 호텔  부산광역시  부산진구   AGODA  2023   12    2   6.80000    1.0
1  아르반 호텔  부산광역시  부산진구   AGODA  2023   12    3   7.26667    3.0
2  아르반 호텔  부산광역시  부산진구   AGODA  2023   12    4  10.00000    1.0
3  아르반 호텔  부산광역시  부산진구   AGODA  2023   12    5   9.80000    2.0
4  아르반 호텔  부산광역시  부산진구   AGODA  2023   12    7   9.50000    2.0

## DB 변수 설명
- 숙박업명: 숙박업체 이름
- 시도명: 숙박업체가 위치한 시·도명
- 구군명: 숙박업체가 위치한 구·군명
- 온라인여행사명: 리뷰·평점이 수집된 플랫폼명
    - 총 8개: GOODCHOICE, YANOLJA, BOOKING, AGODA, EXPEDIA, HOTELS, INTERPARK, DAILY
- 기준년도 / 기준월 / 기준일: 집계 기준일(리뷰·평점 데이터가 수집된 날짜)
- 평점값: 해당 기준일에 등록된 리뷰들의 평균 평점(범위 1–10, 소수점 포함)
- 리뷰등록수: 해당 기준일 신규 리뷰 건수(정수, 현재 데이터상 1–54, 최소 1은 0건 미수집 건 보정값)

In [25]:
df.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 501683 entries, 0 to 501682
Data columns (total 9 columns):
 #   Column   Non-Null Count   Dtype  
---  ------   --------------   -----  
 0   숙박업명     501683 non-null  object 
 1   시도명      501683 non-null  object 
 2   구군명      501683 non-null  object 
 3   온라인여행사명  501683 non-null  object 
 4   기준년도     501683 non-null  int64  
 5   기준월      501683 non-null  int64  
 6   기준일      501683 non-null  int64  
 7   평점값      501683 non-null  float64
 8   리뷰등록수    501683 non-null  float64
dtypes: float64(2), int64(3), object(4)
memory usage: 34.4+ MB


## 중복값 제거

- 누락값은 없는 것 확인했으니 중복값 있는지 확인

In [56]:
# ✅ 전체 컬럼 기준 중복 확인 (첫 발생 제외)
중복_개수 = df.duplicated(subset=df.columns, keep='first').sum()
print(f"중복 행 수 (첫 발생 제외): {중복_개수}")

# ✅ 중복된 모든 행(첫 발생 포함) 정렬해서 보기
중복_행 = (
    df[df.duplicated(subset=df.columns, keep=False)]
      .sort_values(by=df.columns.tolist())
      .reset_index(drop=True)
)

중복_행

중복 행 수 (첫 발생 제외): 671


숙박업명    시도명  구군명     온라인여행사명  기준년도  기준월  기준일   평점값  리뷰등록수
0     강화도 솔향기펜션  인천광역시  강화군  GOODCHOICE  2024    9   24  10.0    1.0
1     강화도 솔향기펜션  인천광역시  강화군  GOODCHOICE  2024    9   24  10.0    1.0
2     강화도 솔향기펜션  인천광역시  강화군  GOODCHOICE  2024    9   30  10.0    1.0
3     강화도 솔향기펜션  인천광역시  강화군  GOODCHOICE  2024    9   30  10.0    1.0
4     강화도 솔향기펜션  인천광역시  강화군  GOODCHOICE  2024   10    6  10.0    1.0
...         ...    ...  ...         ...   ...  ...  ...   ...    ...
1337    호텔리베라거제   경상남도  거제시     YANOLJA  2024    9   18   8.0    1.0
1338    호텔리베라거제   경상남도  거제시     YANOLJA  2024    9   19   6.0    1.0
1339    호텔리베라거제   경상남도  거제시     YANOLJA  2024    9   19   6.0    1.0
1340    호텔리베라거제   경상남도  거제시     YANOLJA  2024   11   25  10.0    1.0
1341    호텔리베라거제   경상남도  거제시     YANOLJA  2024   11   25  10.0    1.0

[1342 rows x 9 columns]

In [57]:
# ✅ 모든 컬럼 기준으로 첫 발생만 유지하고 중복 제거
df = df.drop_duplicates(subset=df.columns, keep='first').reset_index(drop=True)

In [58]:
df.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 501012 entries, 0 to 501011
Data columns (total 9 columns):
 #   Column   Non-Null Count   Dtype  
---  ------   --------------   -----  
 0   숙박업명     501012 non-null  object 
 1   시도명      501012 non-null  object 
 2   구군명      501012 non-null  object 
 3   온라인여행사명  501012 non-null  object 
 4   기준년도     501012 non-null  int64  
 5   기준월      501012 non-null  int64  
 6   기준일      501012 non-null  int64  
 7   평점값      501012 non-null  float64
 8   리뷰등록수    501012 non-null  float64
dtypes: float64(2), int64(3), object(4)
memory usage: 34.4+ MB


In [59]:
# 데이터 정제 전 복사본 생성
df1 = df.copy()

In [60]:
df1

숙박업명    시도명   구군명     온라인여행사명  기준년도  기준월  기준일       평점값  리뷰등록수
0           아르반 호텔  부산광역시  부산진구       AGODA  2023   12    2   6.80000    1.0
1           아르반 호텔  부산광역시  부산진구       AGODA  2023   12    3   7.26667    3.0
2           아르반 호텔  부산광역시  부산진구       AGODA  2023   12    4  10.00000    1.0
3           아르반 호텔  부산광역시  부산진구       AGODA  2023   12    5   9.80000    2.0
4           아르반 호텔  부산광역시  부산진구       AGODA  2023   12    7   9.50000    2.0
...            ...    ...   ...         ...   ...  ...  ...       ...    ...
501007  제천 청풍유스호스텔   충청북도   제천시     YANOLJA  2024   11   15   8.00000    1.0
501008  제천 청풍유스호스텔   충청북도   제천시     YANOLJA  2024   11   22  10.00000    1.0
501009     괴산 더진펜션   충청북도   괴산군     YANOLJA  2024   11   24  10.00000    1.0
501010     괴산 더진펜션   충청북도   괴산군     YANOLJA  2024   11   30  10.00000    1.0
501011    괴산 더좋은하루   충청북도   괴산군  GOODCHOICE  2024   11    4   5.00000    1.0

[501012 rows x 9 columns]

## 기준일 변수 삭제 -> 월 단위로 합치기

[월 집계 결정] 
- 하루 단위 변동을 줄이고 비교·분석을 단순화하기 위해 월 단위로 평균·합계를 집계
- 집계 키: 숙박업명, 시도명, 구군명, 온라인여행사명, 기준년도, 기준월
- 평점값: 해당 월의 일별 평점 단순평균 → 단기 변동 완화 및 추세 파악 용이
- 리뷰등록수: 해당 월 리뷰 수 합계 → 월간 활동량 대표 지표 확보
- 기준일: 제거


In [61]:
# [월 집계: df1을 월 단위로 덮어쓰기]
# - 키: 숙박업명, 시도명, 구군명, 온라인여행사명, 기준년도, 기준월
# - 평점값: 월 단순평균
# - 리뷰등록수: 월 합계
# - 기준일: 제거

keys = ['숙박업명','시도명','구군명','온라인여행사명','기준년도','기준월']

df1 = (
    df1
      .drop(columns=['기준일'], errors='ignore')            # 1) 일자 컬럼 제거
      .groupby(keys, as_index=False)                        # 2) 숙소/지역/OTA/연-월 단위 묶음
      .agg(평점값=('평점값','mean'),                        #    평점: 단순평균
           리뷰등록수=('리뷰등록수','sum'))                  #    리뷰수: 합계
      .assign(평점값=lambda d: d['평점값'].round(3))        # 3) 보기 좋게 평점 반올림(소수 3자리)
      .sort_values(keys, kind='stable')                     # 4) 키 기준 정렬
      .reset_index(drop=True)                               # 5) 인덱스 리셋
)

In [62]:
df1

숙박업명    시도명  구군명  온라인여행사명  기준년도  기준월    평점값  리뷰등록수
0      (주)서울렉스호텔  서울특별시   중구    AGODA  2024    8  7.967    7.0
1      (주)서울렉스호텔  서울특별시   중구    AGODA  2024    9  8.220   11.0
2      (주)서울렉스호텔  서울특별시   중구    AGODA  2024   10  7.614   15.0
3      (주)서울렉스호텔  서울특별시   중구    AGODA  2024   11  7.200   10.0
4      (주)서울렉스호텔  서울특별시   중구  EXPEDIA  2023   12  8.000    1.0
...          ...    ...  ...      ...   ...  ...    ...    ...
63933       힐튼부산  부산광역시  기장군  YANOLJA  2024    6  9.569   24.0
63934       힐튼부산  부산광역시  기장군  YANOLJA  2024    7  9.750    6.0
63935       힐튼부산  부산광역시  기장군  YANOLJA  2024    9  9.471   19.0
63936       힐튼부산  부산광역시  기장군  YANOLJA  2024   10  9.400   13.0
63937       힐튼부산  부산광역시  기장군  YANOLJA  2024   11  9.769   15.0

[63938 rows x 8 columns]

In [24]:
df1['온라인여행사명'].value_counts()

온라인여행사명
GOODCHOICE    18671
YANOLJA       15836
BOOKING       10378
AGODA          8504
EXPEDIA        7443
HOTELS         3073
INTERPARK        26
DAILY             7
Name: count, dtype: int64

In [81]:
df1['기준년도'].value_counts()

기준년도
2024    59866
2023     4072
Name: count, dtype: int64

In [79]:
df1['기준월'].value_counts()

기준월
9     9804
10    9666
11    9265
12    4072
1     4032
3     4018
5     3992
4     3938
2     3914
6     3870
8     3743
7     3624
Name: count, dtype: int64

## 기준년도 변수 삭제

[기준년도 삭제 결정]
- df_2024_6: 2023년 12월 ~ 2024년 5월
- df_2024_12: 2024년 6월 ~ 11월
- 두 데이터 합쳐진 df1: 2023년 12월 ~ 2024년 11월 (12개월)
- 2023년도는 12월만 존재 → 2024년과 연속된 12개월 기간이므로 연도 구분의 분석적 의미가 없음
- 분석은 월 단위로 진행, '기준년도' 변수 삭제

In [63]:
# 기준년도 컬럼 제거
df1 = df1.drop(columns=['기준년도'])

In [64]:
df1

숙박업명    시도명  구군명  온라인여행사명  기준월    평점값  리뷰등록수
0      (주)서울렉스호텔  서울특별시   중구    AGODA    8  7.967    7.0
1      (주)서울렉스호텔  서울특별시   중구    AGODA    9  8.220   11.0
2      (주)서울렉스호텔  서울특별시   중구    AGODA   10  7.614   15.0
3      (주)서울렉스호텔  서울특별시   중구    AGODA   11  7.200   10.0
4      (주)서울렉스호텔  서울특별시   중구  EXPEDIA   12  8.000    1.0
...          ...    ...  ...      ...  ...    ...    ...
63933       힐튼부산  부산광역시  기장군  YANOLJA    6  9.569   24.0
63934       힐튼부산  부산광역시  기장군  YANOLJA    7  9.750    6.0
63935       힐튼부산  부산광역시  기장군  YANOLJA    9  9.471   19.0
63936       힐튼부산  부산광역시  기장군  YANOLJA   10  9.400   13.0
63937       힐튼부산  부산광역시  기장군  YANOLJA   11  9.769   15.0

[63938 rows x 7 columns]

## 온라인여행사명 선택

In [65]:
df1['온라인여행사명'].value_counts()

온라인여행사명
GOODCHOICE    18671
YANOLJA       15836
BOOKING       10378
AGODA          8504
EXPEDIA        7443
HOTELS         3073
INTERPARK        26
DAILY             7
Name: count, dtype: int64

- 동일 숙소 비교라면, 같은 숙소가 여러 OTA에 모두 존재해야 공정하게 비교 가능
- 현재는 OTA별 데이터 개수가 크게 차이남 → 일부 OTA는 많은 숙소 보유, 일부는 극소량(예: DAILY 7건)
- 이런 상태에서 단순 평균·합계를 비교하면 샘플 수 차이로 인한 왜곡 발생

In [66]:
# YANOLJA, GOODCHOICE가 있는 숙소만 추출
target_df = df1[df1['온라인여행사명'].isin(['YANOLJA', 'GOODCHOICE'])]

# 숙박업명으로 그룹화해서 두 여행사 모두 있는지 체크
both_exist = (
    target_df.groupby('숙박업명')['온라인여행사명']
    .nunique()
    .reset_index()
)

# 두 여행사 모두 있는 숙소만 필터
both_exist = both_exist[both_exist['온라인여행사명'] == 2]

# 전체 숙소 수
total_hotels = df1['숙박업명'].nunique()

# 개수와 비율 확인
count_both = len(both_exist)
ratio_both = count_both / total_hotels * 100

print(f"전체 숙소 수: {total_hotels}개")
print(f"두 여행사 모두 있는 숙소 개수: {count_both}개")
print(f"전체 숙소 대비 비율: {ratio_both:.2f}%")

전체 숙소 수: 4905개
두 여행사 모두 있는 숙소 개수: 2630개
전체 숙소 대비 비율: 53.62%


In [67]:
# YANOLJA, GOODCHOICE, AGODA가 있는 숙소만 추출
target_df = df1[df1['온라인여행사명'].isin(['YANOLJA', 'GOODCHOICE','AGODA'])]

# 숙박업명으로 그룹화해서 세 여행사 모두 있는지 체크
three_exist = (
    target_df.groupby('숙박업명')['온라인여행사명']
    .nunique()
    .reset_index()
)

# 세 여행사 모두 있는 숙소만 필터
three_exist = three_exist[three_exist['온라인여행사명'] == 3]

# 전체 숙소 수
total_hotels = df1['숙박업명'].nunique()

# 개수와 비율 확인
count_three = len(three_exist)
ratio_three = count_three / total_hotels * 100

print(f"전체 숙소 수: {total_hotels}개")
print(f"세 여행사 모두 있는 숙소 개수: {count_three}개")
print(f"전체 숙소 대비 비율: {ratio_three:.2f}%")

전체 숙소 수: 4905개
세 여행사 모두 있는 숙소 개수: 948개
전체 숙소 대비 비율: 19.33%


[OTA 평점비교 EDA – 표본 정의 및 선택]

- 목표: 동일 숙소 기준으로 OTA(온라인 여행사) 간 평점·리뷰 차이 비교
- 문제 인식: 숙소별로 보유 OTA가 다르기 때문에, 동일 숙소가 아닌 비교는 편향 발생 가능
- 데이터 현황
- 전체 숙소 수: 4,905개
  - 교집합 규모:
    - YANOLJA & GOODCHOICE 둘 다 있는 숙소: 2,630개
    - YANOLJA & GOODCHOICE & AGODA 모두 있는 숙소: 948개 (전체의 ~19%)

- 의사결정
  - 분석 대상: YANOLJA·GOODCHOICE 두 OTA 모두 존재하는 숙소
  - AGODA까지 포함 시 표본이 948개로 급감 → 대표성 저하 우려로 제외

In [68]:
# 1) YANOLJA, GOODCHOICE만 남김
df1 = df1[df1['온라인여행사명'].isin(['YANOLJA', 'GOODCHOICE'])].copy()

# 2) 같은 숙소(이름+시도명+구군명)에 두 OTA 모두 있는 경우만 유지
keys = ['숙박업명', '시도명', '구군명']
df1 = (
    df1[df1.groupby(keys)['온라인여행사명'].transform('nunique') == 2]
      .sort_values(keys + ['온라인여행사명', '기준월', '평점값', '리뷰등록수'])
      .reset_index(drop=True)
)

In [69]:
df1

숙박업명    시도명  구군명     온라인여행사명  기준월     평점값  리뷰등록수
0      (주)호텔롯데 롯데시티 호텔마포  서울특별시  마포구  GOODCHOICE    9   8.400    6.0
1      (주)호텔롯데 롯데시티 호텔마포  서울특별시  마포구  GOODCHOICE   10   9.800    5.0
2      (주)호텔롯데 롯데시티 호텔마포  서울특별시  마포구  GOODCHOICE   11   9.400    5.0
3      (주)호텔롯데 롯데시티 호텔마포  서울특별시  마포구     YANOLJA    9   9.333    8.0
4      (주)호텔롯데 롯데시티 호텔마포  서울특별시  마포구     YANOLJA   10   9.857   11.0
...                  ...    ...  ...         ...  ...     ...    ...
27119               힐튼부산  부산광역시  기장군     YANOLJA    7   9.750    6.0
27120               힐튼부산  부산광역시  기장군     YANOLJA    9   9.471   19.0
27121               힐튼부산  부산광역시  기장군     YANOLJA   10   9.400   13.0
27122               힐튼부산  부산광역시  기장군     YANOLJA   11   9.769   15.0
27123               힐튼부산  부산광역시  기장군     YANOLJA   12  10.000    5.0

[27124 rows x 7 columns]

In [70]:
# 온라인여행사명 영문 → 한글로 통일
df1['온라인여행사명'] = df1['온라인여행사명'].replace({
    'GOODCHOICE': '여기어때',
    'YANOLJA': '야놀자'
})

In [74]:
df1

숙박업명    시도명  구군명 온라인여행사명  기준월     평점값  리뷰등록수
0      (주)호텔롯데 롯데시티 호텔마포  서울특별시  마포구    여기어때    9   8.400    6.0
1      (주)호텔롯데 롯데시티 호텔마포  서울특별시  마포구    여기어때   10   9.800    5.0
2      (주)호텔롯데 롯데시티 호텔마포  서울특별시  마포구    여기어때   11   9.400    5.0
3      (주)호텔롯데 롯데시티 호텔마포  서울특별시  마포구     야놀자    9   9.333    8.0
4      (주)호텔롯데 롯데시티 호텔마포  서울특별시  마포구     야놀자   10   9.857   11.0
...                  ...    ...  ...     ...  ...     ...    ...
27119               힐튼부산  부산광역시  기장군     야놀자    7   9.750    6.0
27120               힐튼부산  부산광역시  기장군     야놀자    9   9.471   19.0
27121               힐튼부산  부산광역시  기장군     야놀자   10   9.400   13.0
27122               힐튼부산  부산광역시  기장군     야놀자   11   9.769   15.0
27123               힐튼부산  부산광역시  기장군     야놀자   12  10.000    5.0

[27124 rows x 7 columns]

## 월 변수 관련 문제 발견

In [75]:
# 숙소별로 월 데이터 개수 확인
month_counts = (
    df1.groupby('숙박업명')['기준월']
       .nunique()  # 고유 월 개수
       .reset_index(name='월_개수')
       .sort_values('월_개수', ascending=False)
)
month_counts 

숙박업명  월_개수
2629                      힐튼부산    12
2627                     힐튼 경주    12
1                     2278 리조트    12
2586               홀리데이인인천송도호텔    12
2583        홀리데이 인 리조트 알펜시아 평창    12
...                        ...   ...
996               부산 송정뉴비치민박펜션     1
906                 보성 객산카라반펜션     1
2613                   화천 랜드펜션     1
2595  홍천 리버카라반글램핑펜션 (비발디파크 인근)     1
2596              홍천 몰디브(오션월드)     1

[2630 rows x 2 columns]

In [76]:
month_counts['월_개수'].value_counts().sort_index(ascending=False)

월_개수
12     735
11      83
10      28
9       25
8       16
7       14
6        9
5        6
4       22
3     1408
2      232
1       52
Name: count, dtype: int64

- 데이터 정제 후, 월별 데이터 개수가 불균형하게 나타남
- 일부 월은 데이터가 매우 적어(YA·여기어때 교집합 숙소 기준), 월 단위 직접 비교 시 표본 편향 우려
- 이 문제를 해결하기 위해 두 가지 접근 방안을 검토

1. 월 변수 삭제 후 평점 차이만 비교
- 방법: 기준월 제거 → 숙소·지역 동일 기준에서 OTA별 평균 평점 차이 계산
- 장점: 데이터 손실 최소화, 표본 수 최대 확보/ 단순하고 해석이 쉬움
- 단점: 계절성·시즌별 차이 반영 불가/ 시간 흐름에 따른 패턴 분석 불가

2. 성수기·비성수기 파생 변수 생성
- 방법: 월 변수 유지 → 성수기·비성수기 분류 후 비교
- 장점: 계절성과 시즌별 플랫폼 차이 분석 가능/ 마케팅·리뷰 활성화 전략에 활용 가능
- 단점: 표본 수 감소(특히 데이터가 적은 월은 제외됨)/ 일부 시즌 편중 가능성

# 방법 1. 월 변수 삭제

In [90]:
df1.head()

숙박업명    시도명  구군명 온라인여행사명  기준월    평점값  리뷰등록수
0  (주)호텔롯데 롯데시티 호텔마포  서울특별시  마포구    여기어때    9  8.400    6.0
1  (주)호텔롯데 롯데시티 호텔마포  서울특별시  마포구    여기어때   10  9.800    5.0
2  (주)호텔롯데 롯데시티 호텔마포  서울특별시  마포구    여기어때   11  9.400    5.0
3  (주)호텔롯데 롯데시티 호텔마포  서울특별시  마포구     야놀자    9  9.333    8.0
4  (주)호텔롯데 롯데시티 호텔마포  서울특별시  마포구     야놀자   10  9.857   11.0

## 데이터 정제 (월 변수 삭제)

In [78]:
keys = ['숙박업명','시도명','구군명','온라인여행사명']  # ① 집계 기준(숙소+지역+OTA)

df_월제거 = (
    df1.drop(columns=['기준월'])                       # ② 월 변수 제거(월은 안 씀)
       .groupby(keys, as_index=False)                 # ③ 같은 숙소+지역+OTA끼리 묶기
       .agg(평점값=('평점값','mean'),                 # ④ 평점: 평균
            리뷰등록수=('리뷰등록수','sum'))           #    리뷰: 합계
       .assign(평점값=lambda d: d['평점값'].round(3)) # ⑤ 평점 소수 3자리로 정리
       .sort_values(keys, kind='stable')              # ⑥ 키 기준 정렬
       .reset_index(drop=True)                        # ⑦ 인덱스 재설정
)


In [80]:
df_월제거.head()

숙박업명    시도명  구군명 온라인여행사명     평점값  리뷰등록수
0  (주)호텔롯데 롯데시티 호텔마포  서울특별시  마포구     야놀자   9.635   27.0
1  (주)호텔롯데 롯데시티 호텔마포  서울특별시  마포구    여기어때   9.200   16.0
2           2278 리조트   충청남도  보령시     야놀자   9.146   19.0
3           2278 리조트   충청남도  보령시    여기어때   7.472   54.0
4       24게스트하우스 여수점   전라남도  여수시     야놀자  10.000    3.0

## 기술통계 분석

In [ ]:
# EDA 진행을 위해 사이트 야놀자와 여기어때 분리
df_월제거_야놀자 = df_월제거[df_월제거['온라인여행사명'] == '야놀자']
df_월제거_여기어때 = df_월제거[df_월제거['온라인여행사명'] == '여기어때']

In [84]:
df_월제거_야놀자


숙박업명    시도명  구군명 온라인여행사명     평점값  리뷰등록수
0     (주)호텔롯데 롯데시티 호텔마포  서울특별시  마포구     야놀자   9.635   27.0
2              2278 리조트   충청남도  보령시     야놀자   9.146   19.0
4          24게스트하우스 여수점   전라남도  여수시     야놀자  10.000    3.0
6          24게스트하우스 잠실점  서울특별시  송파구     야놀자   9.100   11.0
8             3s 부티크 호텔  인천광역시   서구     야놀자   9.415  494.0
...                 ...    ...  ...     ...     ...    ...
5250            히든베이 호텔   전라남도  여수시     야놀자   9.501  114.0
5252       힐튼 가든인 서울 강남  서울특별시  서초구     야놀자   9.893   42.0
5254              힐튼 경주   경상북도  경주시     야놀자   9.651  439.0
5256           힐튼 호텔 부산  부산광역시  기장군     야놀자   9.619   46.0
5258               힐튼부산  부산광역시  기장군     야놀자   9.600  229.0

[2630 rows x 6 columns]

In [85]:
df_월제거_여기어때

숙박업명    시도명  구군명 온라인여행사명    평점값  리뷰등록수
1     (주)호텔롯데 롯데시티 호텔마포  서울특별시  마포구    여기어때  9.200   16.0
3              2278 리조트   충청남도  보령시    여기어때  7.472   54.0
5          24게스트하우스 여수점   전라남도  여수시    여기어때  9.250    3.0
7          24게스트하우스 잠실점  서울특별시  송파구    여기어때  9.133   23.0
9             3s 부티크 호텔  인천광역시   서구    여기어때  9.449  594.0
...                 ...    ...  ...     ...    ...    ...
5251            히든베이 호텔   전라남도  여수시    여기어때  9.339  142.0
5253       힐튼 가든인 서울 강남  서울특별시  서초구    여기어때  9.427   16.0
5255              힐튼 경주   경상북도  경주시    여기어때  9.488  989.0
5257           힐튼 호텔 부산  부산광역시  기장군    여기어때  9.567   29.0
5259               힐튼부산  부산광역시  기장군    여기어때  9.529  199.0

[2630 rows x 6 columns]

In [88]:
df_월제거_야놀자.describe()

평점값        리뷰등록수
count  2630.000000  2630.000000
mean      9.066938    62.247148
std       1.069993   109.680769
min       2.000000     1.000000
25%       8.811750     5.000000
50%       9.333000    23.000000
75%       9.678750    69.000000
max      10.000000  1242.000000

In [89]:
df_월제거_여기어때.describe()

평점값        리뷰등록수
count  2630.000000  2630.000000
mean      8.895413    58.797338
std       1.026258   102.154968
min       1.000000     1.000000
25%       8.584000     6.000000
50%       9.132000    24.000000
75%       9.500000    63.000000
max      10.000000  1934.000000

<평점>
- 야놀자 평균: 9.067 → 여기어때 평균: 8.895
- 표준편차는 야놀자가 약간 더 큼(1.07 vs 1.03)
- 중앙값(50% 지점)도 야놀자가 약간 더 높음(9.333 vs 9.132)

<리뷰등록수>
- 야놀자 평균: 62.25 → 여기어때 평균: 58.80
- 최댓값은 여기어때가 훨씬 높음(1934 vs 1242) → 일부 숙소에 리뷰 집중 현상
- 사분위수 기준으로는 두 플랫폼이 비슷하나, 야놀자가 하위 구간(25%, 50%)에서 약간 더 낮음

## 대응표본 분석 (동일 숙소 비교)

In [130]:
# 동일 숙소 기준으로 야놀자 vs 여기어때 비교 (개수만)
keys = ['숙박업명','시도명','구군명']

# 두 OTA 모두 있는 숙소만
pair = df1[df1.groupby(keys)['온라인여행사명'].transform('nunique') == 2]

# 가로결합 (멀티인덱스 → 단일 컬럼명 변환)
wide = (
    pair.pivot(index=keys, columns='온라인여행사명', values=['평점값','리뷰등록수'])
        .dropna()
)
wide.columns = [f"{col1}_{col2}" for col1, col2 in wide.columns]
wide = wide.reset_index()

# 평점: 누가 더 높은지
cnt_rating_y = (wide['평점값_야놀자']   > wide['평점값_여기어때']).sum()
cnt_rating_g = (wide['평점값_여기어때'] > wide['평점값_야놀자']).sum()

# 리뷰개수: 누가 더 많은지
cnt_review_y = (wide['리뷰등록수_야놀자']   > wide['리뷰등록수_여기어때']).sum()
cnt_review_g = (wide['리뷰등록수_여기어때'] > wide['리뷰등록수_야놀자']).sum()

print(f"[평점] 야놀자 더 높음: {cnt_rating_y}개 / 여기어때 더 높음: {cnt_rating_g}개")
print(f"[리뷰] 야놀자 더 많음: {cnt_review_y}개 / 여기어때 더 많음: {cnt_review_g}개")


[평점] 야놀자 더 높음: 1592개 / 여기어때 더 높음: 911개
[리뷰] 야놀자 더 많음: 1189개 / 여기어때 더 많음: 1271개


# 방법 2. 성수기·비성수기 파생 변수 생성

In [94]:
df1

숙박업명    시도명  구군명 온라인여행사명  기준월     평점값  리뷰등록수
0      (주)호텔롯데 롯데시티 호텔마포  서울특별시  마포구    여기어때    9   8.400    6.0
1      (주)호텔롯데 롯데시티 호텔마포  서울특별시  마포구    여기어때   10   9.800    5.0
2      (주)호텔롯데 롯데시티 호텔마포  서울특별시  마포구    여기어때   11   9.400    5.0
3      (주)호텔롯데 롯데시티 호텔마포  서울특별시  마포구     야놀자    9   9.333    8.0
4      (주)호텔롯데 롯데시티 호텔마포  서울특별시  마포구     야놀자   10   9.857   11.0
...                  ...    ...  ...     ...  ...     ...    ...
27119               힐튼부산  부산광역시  기장군     야놀자    7   9.750    6.0
27120               힐튼부산  부산광역시  기장군     야놀자    9   9.471   19.0
27121               힐튼부산  부산광역시  기장군     야놀자   10   9.400   13.0
27122               힐튼부산  부산광역시  기장군     야놀자   11   9.769   15.0
27123               힐튼부산  부산광역시  기장군     야놀자   12  10.000    5.0

[27124 rows x 7 columns]

## 데이터 정제 (성수기·비성수기 구분)

In [92]:
# df1 복사해서 새로운 데이터프레임 생성
df_시즌 = df1.copy()

# 7, 8, 12, 1월 = 성수기 / 나머지 = 비수기
peak_map = {7: '성수기', 8: '성수기', 12: '성수기', 1: '성수기'}
df_시즌['시즌구분'] = df_시즌['기준월'].apply(lambda x: peak_map.get(x, '비수기'))

In [93]:
df_시즌

숙박업명    시도명  구군명 온라인여행사명  기준월     평점값  리뷰등록수 시즌구분
0      (주)호텔롯데 롯데시티 호텔마포  서울특별시  마포구    여기어때    9   8.400    6.0  비수기
1      (주)호텔롯데 롯데시티 호텔마포  서울특별시  마포구    여기어때   10   9.800    5.0  비수기
2      (주)호텔롯데 롯데시티 호텔마포  서울특별시  마포구    여기어때   11   9.400    5.0  비수기
3      (주)호텔롯데 롯데시티 호텔마포  서울특별시  마포구     야놀자    9   9.333    8.0  비수기
4      (주)호텔롯데 롯데시티 호텔마포  서울특별시  마포구     야놀자   10   9.857   11.0  비수기
...                  ...    ...  ...     ...  ...     ...    ...  ...
27119               힐튼부산  부산광역시  기장군     야놀자    7   9.750    6.0  성수기
27120               힐튼부산  부산광역시  기장군     야놀자    9   9.471   19.0  비수기
27121               힐튼부산  부산광역시  기장군     야놀자   10   9.400   13.0  비수기
27122               힐튼부산  부산광역시  기장군     야놀자   11   9.769   15.0  비수기
27123               힐튼부산  부산광역시  기장군     야놀자   12  10.000    5.0  성수기

[27124 rows x 8 columns]

In [98]:
# 1) 숙소·지역·OTA·성수기구분별 집계 (평점=평균, 리뷰=합계)
agg = (
    df_시즌
    .groupby(['숙박업명','시도명','구군명','온라인여행사명','시즌구분'], observed=True)
    .agg(평점값=('평점값','mean'),
         리뷰등록수=('리뷰등록수','sum'))
    .reset_index()
)

# 2) 성/비수기 피벗
pvt = (
    agg.pivot(index=['숙박업명','시도명','구군명','온라인여행사명'],
              columns='시즌구분',
              values=['평점값','리뷰등록수'])
       .reset_index()
)

# 3) 컬럼명 변경
pvt.columns = ['숙박업명','시도명','구군명','온라인여행사명',
               '비수기_평점','성수기_평점',
               '비수기_리뷰등록수','성수기_리뷰등록수']

# 4) 성수기·비수기 모두 있는 경우만
pvt = pvt.dropna(subset=['비수기_평점','성수기_평점','비수기_리뷰등록수','성수기_리뷰등록수'])

# 5) 같은 숙소·지역에 두 OTA 모두 있는 경우만
mask_two_ota = pvt.groupby(['숙박업명','시도명','구군명'])['온라인여행사명'].transform('nunique') == 2
df_시즌정제 = pvt[mask_two_ota].copy()

# 6) 값 정리
df_시즌정제['비수기_평점'] = df_시즌정제['비수기_평점'].round(3)
df_시즌정제['성수기_평점'] = df_시즌정제['성수기_평점'].round(3)
df_시즌정제['비수기_리뷰등록수'] = df_시즌정제['비수기_리뷰등록수'].astype(int)
df_시즌정제['성수기_리뷰등록수'] = df_시즌정제['성수기_리뷰등록수'].astype(int)

In [99]:
df_시즌정제

숙박업명    시도명  구군명 온라인여행사명  비수기_평점  성수기_평점  비수기_리뷰등록수  성수기_리뷰등록수
2         2278 리조트   충청남도  보령시     야놀자   8.972   9.666         14          5
3         2278 리조트   충청남도  보령시    여기어때   7.891   6.634         38         16
8        3s 부티크 호텔  인천광역시   서구     야놀자   9.374   9.524        370        124
9        3s 부티크 호텔  인천광역시   서구    여기어때   9.400   9.536        426        168
10    ACC DESIGN호텔  광주광역시   동구     야놀자   8.487   9.422         51         18
...            ...    ...  ...     ...     ...     ...        ...        ...
5253  힐튼 가든인 서울 강남  서울특별시  서초구    여기어때   9.892   8.962          9          7
5254         힐튼 경주   경상북도  경주시     야놀자   9.576   9.852        350         89
5255         힐튼 경주   경상북도  경주시    여기어때   9.531   9.402        678        311
5258          힐튼부산  부산광역시  기장군     야놀자   9.629   9.524        200         29
5259          힐튼부산  부산광역시  기장군    여기어때   9.488   9.611        126         73

[1744 rows x 8 columns]

## 기술통계 분석

In [100]:
# EDA 진행을 위해 사이트 야놀자와 여기어때 분리
df_시즌_야놀자 = df_시즌정제[df_시즌정제['온라인여행사명'] == '야놀자']
df_시즌_여기어때 = df_시즌정제[df_시즌정제['온라인여행사명'] == '여기어때']

In [101]:
df_시즌_야놀자

숙박업명      시도명      구군명 온라인여행사명  비수기_평점  성수기_평점  비수기_리뷰등록수  \
2            2278 리조트     충청남도      보령시     야놀자   8.972   9.666         14   
8           3s 부티크 호텔    인천광역시       서구     야놀자   9.374   9.524        370   
10       ACC DESIGN호텔    광주광역시       동구     야놀자   8.487   9.422         51   
12           CS프리미어호텔      경기도  안양시 만안구     야놀자   8.867   8.500         29   
14         CS프리미어호텔서울    서울특별시      관악구     야놀자   9.643   8.556         59   
...               ...      ...      ...     ...     ...     ...        ...   
5248  히든 클리프 호텔 앤 네이처  제주특별자치도     서귀포시     야놀자   9.422   9.524        102   
5250          히든베이 호텔     전라남도      여수시     야놀자   9.502   9.496         82   
5252     힐튼 가든인 서울 강남    서울특별시      서초구     야놀자   9.889   9.905         29   
5254            힐튼 경주     경상북도      경주시     야놀자   9.576   9.852        350   
5258             힐튼부산    부산광역시      기장군     야놀자   9.629   9.524        200   

      성수기_리뷰등록수  
2             5  
8           124  
10           18  
12           19  
14           19  
...         ...  
5248         17  
5250         32  
5252         13  
5254         89  
5258         29  

[872 rows x 8 columns]

In [102]:
df_시즌_여기어때

숙박업명      시도명      구군명 온라인여행사명  비수기_평점  성수기_평점  비수기_리뷰등록수  \
3            2278 리조트     충청남도      보령시    여기어때   7.891   6.634         38   
9           3s 부티크 호텔    인천광역시       서구    여기어때   9.400   9.536        426   
11       ACC DESIGN호텔    광주광역시       동구    여기어때   8.390   7.514         32   
13           CS프리미어호텔      경기도  안양시 만안구    여기어때   7.953   8.398         46   
15         CS프리미어호텔서울    서울특별시      관악구    여기어때   8.199   8.717         45   
...               ...      ...      ...     ...     ...     ...        ...   
5249  히든 클리프 호텔 앤 네이처  제주특별자치도     서귀포시    여기어때   9.321   9.286        175   
5251          히든베이 호텔     전라남도      여수시    여기어때   9.320   9.376         88   
5253     힐튼 가든인 서울 강남    서울특별시      서초구    여기어때   9.892   8.962          9   
5255            힐튼 경주     경상북도      경주시    여기어때   9.531   9.402        678   
5259             힐튼부산    부산광역시      기장군    여기어때   9.488   9.611        126   

      성수기_리뷰등록수  
3            16  
9           168  
11           18  
13           38  
15           28  
...         ...  
5249         61  
5251         54  
5253          7  
5255        311  
5259         73  

[872 rows x 8 columns]

In [103]:
df_시즌_야놀자.describe()

비수기_평점      성수기_평점   비수기_리뷰등록수   성수기_리뷰등록수
count  872.000000  872.000000  872.000000  872.000000
mean     9.112202    9.121208   98.110092   35.522936
std      0.607848    0.767600  117.634567   42.859013
min      5.000000    4.000000    1.000000    1.000000
25%      8.865750    8.833000   25.000000    9.000000
50%      9.238000    9.297000   55.000000   21.000000
75%      9.530500    9.631500  124.000000   43.000000
max     10.000000   10.000000  912.000000  370.000000

In [104]:
df_시즌_여기어때.describe()

비수기_평점      성수기_평점    비수기_리뷰등록수   성수기_리뷰등록수
count  872.000000  872.000000   872.000000  872.000000
mean     8.976906    9.030147    90.425459   43.902523
std      0.702303    0.888460   101.428880   45.169927
min      1.000000    1.000000     1.000000    1.000000
25%      8.699000    8.760750    26.000000   15.000000
50%      9.111500    9.273500    56.500000   33.000000
75%      9.421000    9.557250   126.000000   58.000000
max     10.000000   10.000000  1348.000000  586.000000

[비수기_평점]

야놀자: 평균 9.112, 중앙값 9.238, 표준편차 0.608

여기어때: 평균 8.977, 중앙값 9.111, 표준편차 0.703
→ 야놀자가 평균 +0.135p 높고, 변동폭도 더 작아 점수 일관성 우수.

[성수기_평점]

야놀자: 평균 9.121, 중앙값 9.297, 표준편차 0.767

여기어때: 평균 9.030, 중앙값 9.274, 표준편차 0.888
→ 성수기에도 야놀자가 소폭 우위(+0.091p), 여기어때는 변동성↑.

[비수기_리뷰등록수]

야놀자: 평균 98.1, 중앙값 55, 표준편차 117.63, 최대 912

여기어때: 평균 90.43, 중앙값 56.5, 표준편차 101.43, 최대 1,348
→ 평균은 야놀자가 약간↑. 다만 여기어때는 일부 상위 숙소에 리뷰 집중(최댓값 큼).

[성수기_리뷰등록수] ← 활성화 1순위

야놀자: 평균 35.52, 중앙값 21, 표준편차 42.86, 최대 370

여기어때: 평균 43.90, 중앙값 33, 표준편차 45.17, 최대 586
→ 성수기 리뷰 생성에서 열세(평균 −8.38, 중앙값 −12) → 성수기 집중 보강 필요.

[결론]
두 플랫폼의 평점은 유사(야놀자 소폭 우위) 하나, **리뷰 수 격차(특히 성수기)**가 커 야놀자 리뷰 활성화 전략이 필요.

## 대응표본 분석 (동일 숙소 비교)

In [105]:
from scipy import stats

# 1) 숙소 기준으로 두 DF 병합
merged = df_시즌_야놀자.merge(
    df_시즌_여기어때,
    on=['숙박업명', '시도명', '구군명'],
    suffixes=('_야놀자', '_여기어때')
)

# 2) 누가 더 높은지 개수 세기
metrics = ['비수기_평점', '성수기_평점', '비수기_리뷰등록수', '성수기_리뷰등록수']
count_results = {}

for m in metrics:
    y_count = (merged[f"{m}_야놀자"] > merged[f"{m}_여기어때"]).sum()
    g_count = (merged[f"{m}_야놀자"] < merged[f"{m}_여기어때"]).sum()
    count_results[m] = {'야놀자_우위': y_count, '여기어때_우위': g_count}

# 3) 대응표본 t-검정
ttest_results = {}
for m in metrics:
    t, p = stats.ttest_rel(merged[f"{m}_야놀자"], merged[f"{m}_여기어때"])
    ttest_results[m] = {'t값': t, 'p값': p}

# 4) 결과 출력
print("=== 개수 비교 ===")
for m, res in count_results.items():
    print(f"{m} → 야놀자 우위: {res['야놀자_우위']} / 여기어때 우위: {res['여기어때_우위']}")

print("\n=== 대응표본 t-검정 ===")
for m, res in ttest_results.items():
    print(f"{m} → t={res['t값']:.3f}, p={res['p값']:.4f}")


=== 개수 비교 ===
비수기_평점 → 야놀자 우위: 546 / 여기어때 우위: 326
성수기_평점 → 야놀자 우위: 470 / 여기어때 우위: 388
비수기_리뷰등록수 → 야놀자 우위: 411 / 여기어때 우위: 444
성수기_리뷰등록수 → 야놀자 우위: 264 / 여기어때 우위: 590

=== 대응표본 t-검정 ===
비수기_평점 → t=5.940, p=0.0000
성수기_평점 → t=2.731, p=0.0064
비수기_리뷰등록수 → t=2.633, p=0.0086
성수기_리뷰등록수 → t=-6.622, p=0.0000


1. 개수 비교
- 평점: 비수기·성수기 모두 야놀자가 여기어때보다 더 높은 숙소 개수가 많음
- 비수기 리뷰등록수: 야놀자 우위 411개, 여기어때 우위 444개 → 여기어때가 약간 더 많은 숙소 보유
- 성수기 리뷰등록수: 야놀자 우위 264개, 여기어때 우위 590개 → 성수기 리뷰 수는 여기어때가 압도적으로 우위

2. 대응표본 t-검정 (평균 차이 검정)
- 비수기 평점: t=5.94, p<0.001 → 야놀자가 평균적으로 유의하게 높음
- 성수기 평점: t=2.73, p=0.0064 → 야놀자가 평균적으로 유의하게 높음
- 비수기 리뷰등록수: t=-2.63, p=0.0086 → 평균적으로 여기어때가 유의하게 많음
- 성수기 리뷰등록수: t=-6.62, p<0.001 → 평균적으로 여기어때가 유의하게 많음

# 최종 결론

1. 분석 개요
- 기간: 2023년 12월 ~ 2024년 11월
- 대상: 동일 숙소 기준 야놀자 vs 여기어때
- 비교 항목: 평점, 리뷰 수

2. 평점
- 동일 숙소 비교 시, 전반적으로 야놀자가 여기어때보다 평점이 높게 나타남
- 그러나 동일 숙소임에도 평점이 높다는 것은 단순히 긍정적인 신호로 보기 어려움
- 예를 들어 동일 숙소에서 야놀자는 5점, 여기어때는 4점을 받았다면, 소비자는 여기어때 리뷰를 더 신뢰할 가능성이 높음
- 이는 야놀자의 평점이 다소 관대한 경향을 시사하며, 고품질·깐깐한 리뷰 활성화 전략이 필요하다.

3. 리뷰 수
- 리뷰 수는 전반적으로 야놀자가 부족했으며, 특히 메인 시즌인 성수기에서 격차가 크게 벌어졌다.
- OTA 경쟁력 강화를 위해 리뷰 수 확대 전략, 특히 성수기 집중 리뷰 확보가 시급하다.

4. 종합 전략 제언
- 평점 관리 고도화: 리뷰 가이드라인 강화 및 품질 높은 후기 확보 → 평점 신뢰도 제고
- 리뷰 수 확대: 리뷰 작성 리워드·이벤트 강화, 참여 유도 최적화
- 성수기 집중 공략: 여행 수요가 몰리는 성수기에 리뷰 확보 속도를 높여 경쟁 우위 확보